src/main/java/io/supertokens/webserver/api/totp/RemoveTotpDeviceAPI.java

```diff
@@ -48,14 +52,25 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
         JsonObject result = new JsonObject();
 
         try {
-            // This step is required only because user_last_active table stores supertokens internal user id.
-            // While sending the usage stats we do a join, so totp tables also must use internal user id.
```

Comment:
write comment if it's app specific or not

Reply:
Done